x/staking/simulation/operations.go

```diff
@@ -345,7 +345,7 @@ func SimulateMsgUndelegate(ak types.AccountKeeper, bk types.BankKeeper, k keeper
 		}
 		// if simaccount.PrivKey == nil, delegation address does not exist in accs. Return error
 		if simAccount.PrivKey == nil {
-			return simtypes.NoOpMsg(types.ModuleName, msg.Type(), "account private key is nil"), nil, fmt.Errorf("delegation addr: %s does not exist in simulation accounts", delAddr)
+			return simtypes.NoOpMsg(types.ModuleName, msg.Type(), "account private key is nil"), nil, nil
```

Comment:
Please give some more description about this.

Reply:
When a module account generated by several other modules among simulation is registered as a validator by CreateValidator Operation of the staking module, an error occurs that cannot be signaled because there is no Private Key, and thus the case is corrected to not be Error.

Reply:
I understand. That's happening after making changes in other modules, is this right?

Reply:
Right, it occurs when modules of https://github.com/cosmosquad-labs/squad perform simulation together, especially liquidstaking.